packages/dev/core/src/Materials/Node/Blocks/PBR/clearCoatBlock.ts

```diff
@@ -43,6 +43,7 @@ export class ClearCoatBlock extends NodeMaterialBlock {
         this.registerInput("tintAtDistance", NodeMaterialBlockConnectionPointTypes.Float, true, NodeMaterialBlockTargets.Fragment);
         this.registerInput("tintThickness", NodeMaterialBlockConnectionPointTypes.Float, true, NodeMaterialBlockTargets.Fragment);
         this.registerInput("worldTangent", NodeMaterialBlockConnectionPointTypes.Vector4, true);
+        this.registerInput("worldNormal", NodeMaterialBlockConnectionPointTypes.Vector3, true);

```

Comment:
I think for consistency you should use `Vector4` instead as the `worldNormal` input in `Light`, `PerturbNormal` and `PBRMetallicRoughness` blocks is `Vector4`.

Reply:
Nice !!! and done :-)